src/ui/common/src/components/integrations/dialogs/IntegrationFileUploadField.tsx

```diff
@@ -123,7 +123,7 @@ export const IntegrationFileUploadField: React.FC<
     const overlay = dragging && theme.palette.gray[100];
     const styling = {
       margin: '16px',
-      height: '16ch',
+      padding: '16px',
```

Comment:
out of curiosity, how tall is this field?

Reply:
This is the styling for the dotted box around the "Upload the CSV file" text.

<img width="662" alt="image" src="https://user-images.githubusercontent.com/30596854/193090371-4a5c6100-03fc-46bd-ad9f-b5dce47e3a35.png">